### PR TITLE
Update helloworld.c

### PR DESCRIPTION
**Pull request by Narutobapu**
Change from `void main` to `int main`

### CODE REVIEW

**Review by ravikush11 — COMMENTED** [review]
Change printf statement to "hello world"

**Review by ravikush11 — CHANGES_REQUESTED** [review]
Change printf statement to "hello world"

**Review by ravikush11 — APPROVED** [review]

File: helloworld.c [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #include<stdio.h>
-void main()
+int main()
```
Comment (ravikush11) [inline helloworld.c]: change to void again

Comment (Narutobapu) [reply helloworld.c]: will do it.

File: helloworld.c [hunk 2]
```diff
@@ -1,5 +1,6 @@
...
 {
 	printf("Hello World\n");
+	return 0;
```
Comment (ravikush11) [inline helloworld.c]: also remove return statement.

Comment (Narutobapu) [reply helloworld.c]: Ok.

**Review by ravikush11 — COMMENTED** [review]
looks ok

**Pull request merged**